### PR TITLE
bump etcd-client to 0.11.1

### PR DESCRIPTION
**Pull request by t-nelson**
#### Problem

old etcd-client dragging in old deps with rustsec advisories

#### Summary of Changes

turn it up to 11

### CODE REVIEW

**Comment by t-nelson** [comment]
hmm... i guess the ga runners don't have protoc? :thinking:

File: .github/workflows/downstream-project-spl.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ jobs:
           source .github/scripts/downstream-project-spl-common.sh
 
           sudo apt update
-          sudo apt install libudev-dev binutils-dev libunwind-dev -y
+          sudo apt install libudev-dev binutils-dev libunwind-dev protobuf-compiler -y
```
Comment (t-nelson) [inline .github/workflows/downstream-project-spl.yml:51]: cc/ @yihau

kinda surprised we didn't have this dragged in already :thinking:

Comment (yihau) [reply .github/workflows/downstream-project-spl.yml:51]: yeah. this script is steal from https://github.com/solana-labs/solana-program-library/blob/master/ci/install-build-deps.sh

btw, the L99 may need your help as well 😞 
https://github.com/solana-labs/solana/blob/8970a3751a4de7ac7656af2e0921bc09b34e402e/.github/workflows/downstream-project-spl.yml#L99

Comment (yihau) [reply .github/workflows/downstream-project-spl.yml:51]: btw, I'm installing this one to docker image.

Comment (yihau) [reply .github/workflows/downstream-project-spl.yml:51]: uploaded and check1 just passed 🎉 

Screenshot 2023-08-24 at 1 21 23 AM

Comment (yihau) [reply .github/workflows/downstream-project-spl.yml:51]: the ci passed on Buildkite side! I think we can ship this one when we have the same update to line 99 :trollface:

Comment (t-nelson) [reply .github/workflows/downstream-project-spl.yml:51]: ah missed that one!  should probably find a common home to specify system dependencies :thinking:

**Review by yihau — APPROVED** [review]
🪖

**Pull request merged**